### ISSUE TITLE
Build issue `g_usermissile.c` Syntax error

### ISSUE THREAD

**Issue by asl97** [zeq2lite/zeq2lite#1]
The [current head](https://github.com/zeq2lite/zeq2lite/commit/24d34a330187ea5b78e671d11397864bfef63868) just throw some weird error when I try to build it on linux.

```
cpp: Game/Game/g_usermissile.c:1 Syntax error in #include
Makefile:2041: recipe for target 'Build/Release-linux-x86_64/Base/Game/g_usermissile.asm' failed
make[2]: *** [Build/Release-linux-x86_64/Base/Game/g_usermissile.asm] Error 1
```

---

It seem like an encoding problem introduce by https://github.com/zeq2lite/zeq2lite/commit/d6a7ebf12b1526eb30135d6879e82bef7e74f96f

throwing the whole directory into `dos2unix` using `cd Game/Game/; dos2unix *` seem to fix the issue.

patch: [patch.txt](https://github.com/zeq2lite/zeq2lite/files/340288/patch.txt)

**Comment by JimmyMD** [comment]
Indeed. that is the line ending bug. i must have uploaded a faulty set of files if that was not fixed yet.

**Comment by sodomon2** [comment]
I did not understand much the patch because it is a .txt and not a .diff or .patch but the error also happens to me

**Comment by sodomon2** [comment]
after executing the command dos2unix the game compil without errors pe ro tells me this when executing the game
